packages/web-components/src/components/rux-status/rux-status.tsx

```diff
@@ -39,7 +39,9 @@ export class RuxStatus {
     }
 
     connectedCallback() {
-        if (this.status) this.validateStatus(this.status)
+        this.status
```

Comment:
Because this is changing the status internally, we get a console warning from stencil: 
```
@Prop() "status" on <rux-status> is immutable but was modified from within the component.
```
Could we instead change the status prop to have a default of 'normal'? I'll make a comment at that location, too.

Reply:
Welp since github won't let me comment on it since it's not a 'changed file', it's on line 19. If we do: 
```
@Prop({ reflect: true }) status?: Status = 'normal'
```
then the situation of having no status passed in is solved before render and we won't get that Stencil warning. 
Any downsides to this that I may have missed?